### ISSUE TITLE
Is it possible to adapt the web pages to mobile devices?

### ISSUE THREAD

**Issue by CS123n** [tensorflow/tensorboard#5910]
Please describe the problem that you’re facing and the enhancements that
you’d like to see. Feel free to include screenshots or code samples.

Hi, I use the tensorboard on my phone sometimes, but it is not very easy to read the results.

**Comment by arcra** [comment]
Hello,

I have a few questions, to clarify the use case:
1) Do you start a TB instance on a computer, and then you access the web app from your phone while connected to the same network?
2) Are there any specific features you'd like to see from your phone? Or any that are particularly bad when using your phone? or is it just that it's generally not mobile-friendly?

I don't think we were expecting users to see the TB UI from their phones, so likely not a big priority for us. It would be helpful to understand the use case and/or the benefit of accessing it on a mobile device.

**Comment by CS123n** [comment]
Yes, I use a ubuntu server to host the TB and see the result on my phone. I use SCALARS and IMAGES in most cases. 
1. When I put my mouse in a certain position, I can see the results of scalars at the same position. However, it is not available on mobile phones.
2. The sidebar is not mobile-friendly, maybe provide a bottom to fold them in most cases for mobile phone users.

I understand this is not popular demand. So just do it when you have time. Thanks a lot.

**Comment by nfelt** [comment]
Duplicate of #1412

**Comment by nfelt** [comment]
This sounds like mostly the same request as the longstanding #1412, so I've marked as a duplicate.
